package.json

```diff
@@ -31,6 +31,7 @@
     "cheerio": "^0.19.0",
     "clean-css": "^3.1.9",
     "cli": "^0.6.6",
+    "filter-css": "0.0.1",
```

Comment:
seems like a dupe dependency here? it told me it was opting for `0.0.1` when I installed